src/setup-gcloud.ts

```diff
@@ -90,6 +95,10 @@ export async function run(): Promise<void> {
       core.info('Successfully exported Default Application Credentials');
     }
   } catch (error) {
-    core.setFailed(error.message);
+    if (error instanceof Error) {
+      core.setFailed(error.message);
+    } else {
+      core.setFailed("'setup-gcloud' failed to be installed.");
+    }
```

Comment:
Cant we just `core.setFailed('setup-gcloud' failed to be installed: ${errror});`